server/routerlicious/packages/services-shared/src/storage.ts

```diff
@@ -274,23 +274,31 @@ export class DocumentStorage implements IDocumentStorage {
 			lumberjackProperties,
 		);
 
+		const document: IDocument = {
+			createTime: Date.now(),
+			deli: JSON.stringify(deli),
+			documentId,
+			session,
+			scribe: JSON.stringify(scribe),
+			tenantId,
+			version: "0.1",
+			storageName,
+			isEphemeralContainer,
+		};
+		const documentDbValue: IDocument & { ttl?: number } = {
+			...document,
+		};
+		if (isEphemeralContainer) {
+			documentDbValue.ttl = 60 * 60 * 24; // 24 hours in seconds
```

Comment:
We can define `60*60*24` as const first

Reply:
I think this is OK. in other places, we use an indecipherable number, which is the result of this multiplication, so I feel this is already more readable than other spots. Maybe worth a follow-up up though.